CODEOWNERS

```diff
@@ -3,4 +3,4 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence, these accounts
 # will be requested for review when someone opens a pull request.
-* @aws/amazon-braket-maintainers
+* @amazon-braket/braket-maintainers
```

Comment:
Does amazon-braket/braket-maintainers actually exist? @krneta

Reply:
Yes, @shpface created this team just today https://github.com/orgs/amazon-braket/teams/braket-maintainers